Sources/Beagle/Sources/Components/ServerDrivenComponent/ServerDrivenComponent.swift

```diff
@@ -57,6 +57,11 @@ extension ServerDrivenComponent {
             )
         }
     }
+    
+    private func getFirstChildContainerId() -> String? {
+        let child = self as? Beagle.Container
```

Comment:
is it correct to assume it will always be a container? Isn't it better to take the identifercomponent interface that has the id property?

Reply:
When it reaches this point it is a Container. Sooner on line 51 I check if it is a Screen. Currently these are the two possibilities to create beagle layouts using our BFF signatures.